server/dex/feemgr.go

```diff
@@ -0,0 +1,81 @@
+package dex
+
+import (
+	"strconv"
+	"sync/atomic"
+
+	"decred.org/dcrdex/server/asset"
+	"decred.org/dcrdex/server/market"
+)
+
+// FeeManager manages fee fetchers and a fee cache.
+type FeeManager struct {
```

Comment:
If you're readying another commit, I'd suggest the following assertions for both documentation and anti-fragility, otherwise not critical:

```go
var _ market.FeeSource = (*FeeManager)(nil)
var _ market.FeeFetcher = (*feeFetcher)(nil)
```